<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>1082</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="pixmap">
          <pixmap>inicio 3.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(206, 208, 221);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>281</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>imagenes login/hospital.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>290</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 20pt &quot;MV Boli&quot;;
border:None;</string>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="entry_usuarios">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>350</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushEdit{
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su correo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>470</y>
         <width>220</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 20pt &quot;MV Boli&quot;;
border:None;</string>
       </property>
       <property name="text">
        <string>Contraseña</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="entry_contrasenia">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>520</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushEdit{
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ingrese su contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="boton_iniciar">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>640</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

	
	
	background-color: rgb(0, 170, 255);
	font: 75 12pt &quot;MS Shell Dig 2&quot;;
	border-radius:10px;
}

QPushButton:hover{

	
	background-color:rgb(85, 170, 255);
	font: 75 12pt &quot;MS Shell Dig 2&quot;;
}</string>
       </property>
       <property name="text">
        <string>Iniciar sesion</string>
       </property>
      </widget>
      <widget class="QPushButton" name="boton_salir">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>770</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{

	
	background-color: rgb(0, 0, 0,0%);
	font: 75 12pt &quot;MS Shell Dig 2&quot;;
	border-radius:10px;
	border:None;
}

QPushButton:hover{ 
	background-color: rgb(85, 170, 255);
	font: 75 12pt &quot;MS Shell Dig 2&quot;;
	
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>imagenes login/cerrar-sesion.png</normaloff>imagenes login/cerrar-sesion.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_usuario_incorrecto">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>410</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 12pt &quot;MV Boli&quot;;
color: rgb(255, 0, 0);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_contrasenia_incorrecta">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>580</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 12pt &quot;MV Boli&quot;;
color: rgb(255, 0, 0);
border:None;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
